docs/tutorial/getting-started.md

```diff
@@ -1,3 +1,210 @@
 # Getting started
 
-TODO
+## 1 Prerequisites
+
+- A Linux machine.
+
+## 2 Install Pebble
+
+### 2.1 Download and install the latest release
+
+1. Go to the [release page](https://github.com/canonical/pebble/releases) of Pebble and download the latest release according to your architecture.
+2. Unzip it. In the unzipped folder, move the `pebble` binary to `/usr/local/bin/` (or any directory that is in your `$PATH`).
+
+> Alternatively, you can find the latest tag on the [latest release page](https://github.com/canonical/pebble/releases/latest), then run the following command to download, unzip and install the latest release (suppose `v0.11.0` is the latest tag and your architecture is `amd64`; replace the tags and arch accordingly):
+> 
+> ```bash
+> $ wget -q https://github.com/canonical/pebble/releases/download/v1.11.0/pebble_v1.11.0_linux_amd64.tar.gz
+> $ tar zxvf pebble_v1.11.0_linux_amd64.tar.gz
+> $ sudo mv pebble /usr/local/bin/
+> ```
+
+### 2.2 Install from source
+
+If you want, you can build and install Pebble from source:
+
+1. Follow the official Go documentation [here](https://go.dev/doc/install) to download and install Go.
+2. After installing, you will want to add the `$GOBIN` directory to your `$PATH` so you can use installed tools. For more information, refer to the [official documentation here](https://go.dev/doc/install/source#environment).
+3. Run `go install github.com/canonical/pebble/cmd/pebble@latest` to build and install Pebble.
+
+### 2.3 Sanity check
+
+After installation, if you run `pebble`, you should see some output equivalent to the following:
+
+```bash
+$ pebble
+Pebble lets you control services and perform management actions on
+the system that is running them.
+
+Usage: pebble <command> [<options>...]
+
+Commands can be classified as follows:
+
+       Run: run, help, version
+      Plan: add, plan
+  Services: services, logs, start, restart, signal, stop, replan
+    Checks: checks, health
+     Files: push, pull, ls, mkdir, rm, exec
+   Changes: changes, tasks
+   Notices: warnings, okay, notices, notice, notify
+
+Set the PEBBLE environment variable to override the configuration directory
+(which defaults to /var/lib/pebble/default). Set PEBBLE_SOCKET to override
+the unix socket used for the API (defaults to $PEBBLE/.pebble.socket).
+
+For more information about a command, run 'pebble help <command>'.
+For a short summary of all commands, run 'pebble help --all'.
+```
+
+Pebble is invoked using `pebble <command>`. To get more information:
+
+- To see a help summary, type `pebble -h`.
+- To see a short description of all commands, type `pebble help --all`.
+- To see details for one command, type `pebble help <command>` or `pebble <command> -h`.
```

Comment:
We should probably pick one or the other. I'd say `pebble help <command>` as that's what the main help output suggests.